functionary/core/apps.py

```diff
@@ -1,6 +1,6 @@
 from django.apps import AppConfig
 
 
-class TaskmanConfig(AppConfig):
-    default_auto_field = 'django.db.models.BigAutoField'
-    name = 'core'
+class CoreConfig(AppConfig):
```

Comment:
Another unrelated item.  Left over reference to the name in the proof of concept phase.